static/js/utils/number_utils.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
can we keep these in js/shared/ instead of adding a new utils folder?

Reply:
right now, the shared folder has constants, utils, types, and various different kinds of files. I think it might make more sense to create this utils folder at this level and start moving files out of the shared folder (seeing how we also have a types folder at this level). I think the name shared doesn't tell us much about what's in the folder and if we were to clean up the contents of that folder into subfolders, that will create unnecessary additional nesting. What do you think?

Reply:
ok, though we might want a more specific name than utils (maybe data_utils? or chart_utils)? let's keep in mind that want to avoid getting it to be too much of a grab bag of functionality.

Reply:
maybe we can keep it like this for now and as more files get added to the folder, we'll have a better idea of how to split it up?

Reply:
sure